modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -347,51 +347,15 @@ aws ec2 create-vpc-endpoint \
 
 == Access Redpanda services through VPC endpoint
```

Comment:
I missed this in my review, but while we can reuse the ports table, we can't reuse the rest of the text, because
- PrivateLink is an AWS product, while PSC is GCP's.
- "VPC endpoint" is AWS terminology. In the context of PSC, it's called a "PSC endpoint".